pkg/defaults/defaults.go

```diff
@@ -74,7 +74,7 @@ const (
 	DefaultEVEHV                 = "kvm"
 	DefaultQemuCpus              = 4
 	DefaultQemuMemory            = 4096
-	DefaultEVESerial             = "31415926"
+	DefaultEVESerial             = "*"
```

Comment:
We do not need to modify defaults here. We have this logic inside `Config` function of DevModel.